cl/opinion_page/views.py

```diff
@@ -30,9 +30,19 @@
 from cl.search.models import Docket, OpinionCluster, RECAPDocument
 
 
-@ratelimit_if_not_whitelisted
 def view_docket(request, pk, slug):
     docket = get_object_or_404(Docket, pk=pk)
+    return view_docket_subr(request, docket)
```

Comment:
Sorry if I wasn't clear on this, but I actually *prefer* a redirect here rather than a rewrite. I only want one URL to go to this location, and I view this as a lookup feature, not an alternate URL.

Reply:
Err, I assume this is not a comment on this portion of the code.